### DISCUSSION TITLE
Reverse Proxy - Modify requests

### DISCUSSION THREAD

**Discussion by RjRDRG** [abhinavsingh/proxy.py#1218] [Q&A] [answered]
Hello, 

Is it possible to modify a request body and headers with the reverse proxy plugin?

**Comment by abhinavsingh** [comment, marked as answer]
@RjRDRG Thanks for raising this.  There is no way currently.  But is easy to add this feature, since we already have a plugin based functionality to modify proxy request/responses.  We can add same lifecycles to reverse proxy too and get the same experience.

I have transferred this as an issue to keep track of it for next release.  Best.

**Reply by abhinavsingh** [reply]
@RjRDRG https://github.com/abhinavsingh/proxy.py/pull/1252 will add support for modifying the request object before the request gets routed.  Give it a try when you can.  Let me know if this isn't working as expected.  Best
